proxy/receiver_proxy_test.go

```diff
@@ -169,7 +169,7 @@ func TestMain(m *testing.M) {
 	defer archiveServer.Close()
 
 	for i := range 3 {
-		proxy, err := StartTestOrderflowProxy(fmt.Sprintf("proxy:%d", i))
+		proxy, err := StartTestOrderflowProxy(fmt.Sprintf("proxy:%d", i), "/tmp/orderflow-proxy-test.cert", "/tmp/orderflow-proxy-test.key")
```

Comment:
All 3 proxies here will have the same certifiate and use the same file, this crease a dependency between test runs. 1 test run you read certs, another you read from file.

I think it would be better to use temp files always and have a test that tests certificate after restart explicitly.

Reply:
i know and good point! 

i wanted to do that, but this one test uses `testing.M` and I couldn't figure out how to access `testing.T` for it's temp directory functionality

Reply:
updated